doc/introduction.md

```diff
@@ -0,0 +1,42 @@
+# PCT
+
+PCT (Proton Computed Tomography) is a toolkit used to process proton CT data and reconstruct proton stopping power maps. PCT is written both in C++ and Python, and is designed to be used either as a code library, or with command-line applications.
+
+## Installation
+
+PCT can be installed from source from its [git repository](https://github.com/SimonRit/PCT) hosted at GitHub. The command to clone the repository is:
+```bash
+git clone https://github.com/SimonRit/PCT.git
+```
+
+Configuration and compilation is handled via [CMake](https://cmake.org/). Configuration can be achieved using
+```bash
+cmake path/to/source/folder  # where PCT was cloned
+```
+or using a graphical tool such as `ccmake`. Once configuration completes, the compilation can be achieved using
+```bash
+make
+```
+Keep in mind that `make` can be sped-up using the `-j` option, allowing to parallelize the compilation, for instance using `make -j 4` if your computer has 4 cores.
+
+Optionally, PCT can be added to the user's `$PATH` variable using for instance
+```bash
+export PATH=path/to/build/folder:${PATH}  # where PCT was built
+```
+in the user's `.bashrc` file.
+
+Some functionalities of PCT require [GATE](http://www.opengatecollaboration.org/) to run. [Instructions on how to install GATE can be found in the GATE documentation.](https://opengate-python.readthedocs.io/en/master/user_guide/user_guide_installation.html) An example of proton CT GATE simulation can be found [here](../gate/protonct.py).
```

Comment:
I would remove this but we can wait for the ITK module conversion to write something similar to RTK.

Reply:
Agreed, let's wait until the module is ready.